client/src/components/timeline-window-component/clientWindow.jsx

```diff
@@ -0,0 +1,183 @@
+                "aria-labelledby": "split-button",
+              }}
+              >
+                <MenuItem onClick={handleClickDots} sx={{ justifyContent: "center" }} >
+                  <DownloadIcon sx={{color: "grey", paddingRight: "10px"}}/>
+                  Export as JSON
+                </MenuItem>
+                </Menu>
+          </Grid>
+        </Grid>
+      </Box>
+
+          <CardContent>
+          
+            <ButtonGroup
+              variant="contained"
+              style={{ width: "100%", borderRadius: "8px" }}
+              aria-label="split button"
+              ref={buttonGroupRef}
+              color="primary"
+              sx={{'& .MuiButtonBase-root': {backgroundColor: "#3568E5"}}}
+              
+            >
+              <Button fullWidth startIcon={<AddIcon />}>Add New Entry</Button>
+              <Button onClick={handleClick}>
+                <ArrowDropDownIcon />
+              </Button>
+              <Menu
+              anchorEl={anchorEl}
+              open={open}
+              onClose={handleClose}
+              MenuListProps={{
+                "aria-labelledby": "split-button",
+              }}
+              PaperProps={{
+                style: {
+                  width: buttonGroupRef
+                    ? buttonGroupRef.current?.clientWidth
+                    : null,
+                },
+              }}
+              >
+                <MenuItem onClick={handleClose} sx={{ justifyContent: "center" }} >
+                  Placed Job Seeker
+                </MenuItem>
+                <MenuItem onClick={handleClose} sx={{ justifyContent: "center" }} >
+                  Other Note
+                </MenuItem>
+
+              </Menu>
+            </ButtonGroup>
+
+           
+           
+            <TabContext value={value}>
+              <TabList onChange={handleChange} sx={{
+        '& .MuiTabs-flexContainer': { display: 'flex', width: "100%", gap: -2 }, 
+        '& .MuiButtonBase-root': {padding: 0}
+      }}>
+                <Tab label="ALL" value="1" />
+                <Tab label="CONTACTED" value="2" />
+                <Tab label="UPDATES" value="3" />
+                <Tab label="NOTES" value="4" />
+              </TabList>
+        
+        <Divider/>
+        <TabPanel value="1">ALL</TabPanel>
+        <TabPanel value="2">CONTACTED</TabPanel>
+        <TabPanel value="3">UPDATES</TabPanel>
+        <TabPanel value="4">NOTES</TabPanel>
+        
+      </TabContext>
+
+      
+      
+      
+          
+        </CardContent>
+        <TextField
+      id="search"
+      type="search"
+      placeholder="Search..."
+      value={searchTerm}
+      onChange={handleSearchChange}
+      fullWidth
+      size="small"
+      InputProps={{
+        startAdornment: (
+          <InputAdornment position="start">
+            <SearchIcon style={{ fontSize: '2rem' }}/>
+          </InputAdornment>
+        ),
+      }}
+    />             
```

Comment:
The search component can be removed from each of the timeline windows (It's been listed as a nice to have for now)